pennylane/templates/embeddings/amplitude.py

```diff
@@ -108,9 +48,8 @@ def AmplitudeEmbedding(features, wires, pad_with=None, normalize=False, pad=None
         gradients with respect to the features cannot be computed by PennyLane.
 
     Args:
-        features (tensor-like): input vector of length ``2^n``, or less if `pad_with` is specified
-        wires (Iterable or :class:`.wires.Wires`): Wires that the template acts on.
-            Accepts an iterable of numbers or strings, or a Wires object.
+        features (tensor_like): input tensor of dimension ``(2^n,)``, or less if `pad_with` is specified
+        wires (Iterable): wires that the template acts on
```

Comment:
Updated the docstring everywhere: 

* `Wires` is not user facing (and an iterable anyways), so removed mention
* tensor_like is more accurate than array